src/static/css/pages/beta.less

```diff
@@ -77,6 +72,7 @@
     .respond-to-min(801px, {
         // At this width, the wrapper gains `position: relative`,
         // so the button position has to be adjusted accordingly.
+        // This number depends on something in cf-layout
```

Comment:
Oh, magic numbers. Any idea if we can somehow make this em so it at least changes with the font-size?